seed/static/seed/js/controllers/insights_program_controller.js

```diff
@@ -134,6 +136,27 @@ angular.module('BE.seed.controller.insights_program', [])
             data: {
             },
             options: {
+              onClick: (event) => {
+                var activePoints = event.chart.getActiveElements(event);
+
+                if (activePoints[0]) {
+                  var activePoint = activePoints[0]
+                  cycle_name = $scope.data.graph_data.labels[activePoint.index]
+                  cycle = $scope.cycles.find(c => c.name == cycle_name);
+                  shown_dataset_index = activePoint.datasetIndex;
+
+                  // update locally stored insights_property configs
+                  const property_configs = JSON.parse(localStorage.getItem('insights.property.configs'));
+                  property_configs.compliance_metric_id = $scope.selected_metric;
```

Comment:
I'm getting an error on this line, probably b/c I don't yet have localStorage item configured?

error is: cannot set properties of null (setting 'compliance_metric_id')